report-deployment-status-to-slack/action.yml

```diff
@@ -17,9 +17,6 @@ inputs:
   environment:
     description: "The name of the environment."
     required: true
-  slack-channel:
-    description: "The name of the Slack channel."
-    default: team-eng-qa-devops
```

Comment:
❓ Why remove the parameter here and hard code values later in this workflow, instead of passing the argument from the workflow that calls this?

Reply:
We don't need the parameter ever again? I see that it's used here. https://github.com/bitwarden/mothership/blob/main/.github/workflows/_notify.yml#L37

I would imagine that this action will be used as well for other Global Service environments, where the channel would be very dynamic based on the team ownership of the environment. I think removing this parameter would remove a lot of value from this action.

Reply:
Hey @mimartin12, can you review now?

Reply:
Sure, it looks like just formatting changes then?

Reply:
Yeah